diff --git a/CMakeLists.txt b/CMakeLists.txt
index 38c297395..b8f40fb78 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -104,7 +104,7 @@ if (${CMAKE_BUILD_TYPE} STREQUAL "Debug")
 elseif (${CMAKE_BUILD_TYPE} MATCHES "Rel")
   add_definitions(-DRELEASE)
   if (NOT MSVC)
-    add_compile_options(-O3 $<$<CXX_COMPILER_ID:GNU>:-flto=auto>)
+    add_compile_options(-O3)
   endif()
 else()
   message(FATAL_ERROR "Unknown build type: " ${CMAKE_BUILD_TYPE})
